### PR TITLE
Add Percy integration for demos

### PR DESCRIPTION
**Pull request by JakeChampion**
Adds a new GH Workflow which will run Percy on the demos only if the `wontfix` label is added, it will then comment on the PR when Percy has finished and finally remove the `wontfix` label.

The reason we don't want to run Percy all the time is because Percy charge per screenshot. We currently have 5,000 screenshots a month as we are on the free tier. I figured having an easy manual trigger for Percy would be the best approach.

If we like this idea, we should add a `percy` label to the list of labels anduse that instead of `wontfix`.

### CODE REVIEW

**Comment by notlee** [comment]
I love this idea, nice work Jake! :D But I don't understand using the `won't fix` label? Adding a `percy` label sounds good :)

**Comment by chee** [comment]
@notlee i think the `wontfix` label was only because the label-sync action kept deleting the percy label :D

**Comment by chee** [comment]
Let's have a chat at office hours about auto-mating the adding of the `percy` label based on what files have changed :)

**Comment by origamiserviceuser** [comment]
👋 Percy has finished running the visual regression testing!

File: .github/workflows/percy.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      with:
+        build-directory: "demos/local"
+      env:
+        PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+    - if: contains(github.event.pull_request.labels.*.name, 'percy')
+      uses: actions/github-script@0.9.0
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          github.issues.createComment({
```
[the remark this answers is not in the document]
Comment (chee) [reply .github/workflows/percy.yml:33]: oh nice

**Pull request merged**